pkg/kyma/cmd/provision/minikube/hosts_darwin.go

```diff
@@ -11,18 +16,65 @@ const hostsFile = "/etc/hosts"
 
 const defaultVMDriver = vmDriverHyperkit
 
+func isWithSudo() bool {
+	val := os.Getenv("SUDO_UID")
+	if val != "" {
+		return true
+	}
+	return false
+}
+
+func promptUser() bool {
+	for {
+		fmt.Print("Type [y/n]: ")
+		var res string
+		_, err := fmt.Scanf("%s", &res)
+		if err != nil {
+			return false
+		}
+		switch res {
+		case "yes":
```

Comment:
Is it correct to check for the words "yes" and "no" when the prompt asks for "y" or "n"?
I usually only type "y" in these cases.

Reply:
Fixed